modules/querying-bootstrap-node-journal-logs.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * support/troubleshooting/gathering-cluster-data.adoc
+
+[id="querying-bootstrap-node-journal-logs_{context}"]
+= Querying bootstrap node journal logs
+
+If you experience bootstrap-related issues, you can gather `bootkube.service` `journald` unit logs and container logs from the bootstrap node.
+
+.Prerequisites
+
+* You have SSH access to your bootstrap node.
+* You have the fully qualified domain name of the bootstrap node.
+
+.Procedure
+
+. Query `bootkube.service` `journald` unit logs from a bootstrap node during {product-title} installation. Replace `<cluster_name>.<base_domain>` with appropriate values:
++
+----
+$ ssh core@bootstrap.<cluster_name>.<base_domain> journalctl -b -f -u bootkube.service
```

Comment:
To be more accurate, we better to replace the `bootstrap.<cluster_name>.<base_domain> ` to the bootstrap public FQDN, like `<bootstrap_fqdn>`, not all bootstrap host will get a public hostname like `bootstrap.<cluster_name>.<base_domain> ` .
As we already mentioned in the `Prerequisites `
```
You have the fully qualified domain name of the bootstrap node
```

Reply:
@gpei thank you for your review. I have now resolved this for all bootstrap node command line references throughout the PR.